src/Avalonia.Input/PointerPoint.cs

```diff
@@ -67,9 +81,13 @@ public enum PointerUpdateKind
         LeftButtonPressed,
         MiddleButtonPressed,
         RightButtonPressed,
+        XButton1Pressed,
```

Comment:
That breaks binary compatibility. Not sure if we care at the current point

Reply:
Yeah I don't think we care at this point. 0.9 is going to have a number of breaking changes so everything is going to need to be recompiled anyway.